rules/network/command_and_control_fin7_c2_behavior.toml

```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2020/07/06"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects a known command and control pattern in network events. The FIN7 threat group is known to use this
+command and control technique, while maintaining persistence in their target's network.
+"""
+false_positives = [
+    """
+    This rule could identify benign domains that are formatted similarly to FIN7's command and control algorithm. Alerts
+    should be investigated by an analyst to assess the validity of the individual observations.
+    """,
+]
+index = ["packetbeat-*"]
+language = "lucene"
+license = "Elastic License"
+name = "Possible FIN7 Command and Control Behavior"
```

Comment:
I think we could add a little more specificity to this. If we make another FIN7 C2 rule, then we'll want to differentiate the rules. Maybe adding something about the DGA or DNS-based approach to detection in the title?

Reply:
Agreed, added.